users/tests.py

```diff
@@ -1,27 +1,22 @@
+import faker
 from django.test import TestCase
+
 from users.models import User
 
-import faker
 
 class UserTestCase(TestCase):
-  def setUp(self):
-      self.fake=faker.Faker()
-      self.first_name=self.fake.first_name()
-      self.last_name=self.fake.last_name()
-      self.email=self.fake.email()
-      self.gender='NS'
-
-      User.objects.create(
-          first_name=self.first_name,
-          last_name=self.last_name,
-          email=self.email,
-          gender=self.gender
-      )
+    def setUp(self):
```

Comment:
I know that in this class it's only one unit test.
But if you want to have general tests settings, it's better to use [setUpClass](https://docs.djangoproject.com/en/3.1/topics/testing/tools/#django.test.SimpleTestCase.databases) that is executed once at the beginning.
setUp is executed before every unit test.